lua/wire/client/text_editor/wire_expression2_editor.lua

```diff
@@ -1891,6 +1891,19 @@ function Editor:Setup(nTitle, nLocation, nEditorType)
 	if not useValidator then
 		self.C.Val:SetVisible(false)
 	end
+	
+	-- PickColor on E2
+	local useColorMixer = nEditorType == "E2"
```

Comment:
For consistency this should probably be with `useValidator`, `useSoundBrowser`, and others above.